lib/db/schema/patch-021-022.sql

```diff
@@ -0,0 +1,71 @@
+CREATE TABLE IF NOT EXISTS verificationReminders (
+  id BINARY(16) PRIMARY KEY,
+  uid BINARY(16) NOT NULL,
+  email VARCHAR(255) NOT NULL,
+  emailCode BINARY(16) NOT NULL,
+  type VARCHAR(255) NOT NULL,
+  acceptLanguage VARCHAR(255) NOT NULL,
```

Comment:
Are we able to lookup `emailCode` and `acceptLanguage` via a join with the main account table rather than duplicating them here?